docs/docs/smart-contracts/timelock-controller.md

```diff
@@ -1,13 +1,16 @@
-## Overview
+---
+sidebar_position: 7
+title: Timelock Controller
+---
 
 This example shows how you can reuse the implementation of
 [timelock-controller](https://github.com/Supercolony-net/openbrush-contracts/tree/main/contracts/governance/timelock-controller).
 
-## Steps
+## Step 1: Include dependencies
```

Comment:
We have the same readme in the `example` folder. Could you update it in the same way? Or maybe you can simplify the readme in examples and create a link on readme from the `docs` folder?)

Reply:
Done